src/common/k8s-api/kube-api.ts

```diff
@@ -217,7 +217,7 @@ export class KubeApi<
   readonly kind: string;
   readonly apiVersion: string;
 
-  @observable apiBase: string;
+  @observable apiBase = "";
```

Comment:
This is the fix from earlier, right? Maybe a cleaner way to fix this, without the intermediate empty string, would be to defer `makeObservable` until the real initial string value is set.

Reply:
Sure